swagger/swagger-generator/generator-springmvc/src/main/java/org/apache/servicecomb/swagger/generator/springmvc/processor/annotation/RequestPartAnnotationProcessor.java

```diff
@@ -56,11 +57,34 @@ public HttpParameterType getHttpParameterType(RequestPart parameterAnnotation) {
   @Override
   public void fillParameter(Swagger swagger, Operation operation, FormParameter formParameter, Type type,
       RequestPart requestPart) {
-    Property property = new FileProperty();
-    if (TypeFactory.defaultInstance().constructType(type).isContainerType()) {
-      property = new ArrayProperty(new FileProperty());
-    }
+    Property property = resolveParamProperty(type);
```

Comment:
Seems RequestPartAnnotationProcessor can inherit from AbstractSpringmvcSerializableParameterProcessor

Reply:
Done, please review it.

Reply:
actually I mean fillParameter can call super.fillParameter, because jaxrs works fine by doing this.

Reply:
Done